### DISCUSSION TITLE
Suggestion. Add option to localboot using custom wallpapers

### DISCUSSION THREAD

**Discussion by megaenderdragon** [ventoy/Ventoy#1242] [Ideas]
Just like in the title. Some custom wallpapers don't show localboot, or other options, Like memdisk. Is there a way to show these options in custom wallpapers? Also, can you add an option to switch between wallpapers?

**Comment by ventoy** [comment]
https://www.ventoy.net/en/plugin_theme.html
Add these following hbox definitions to your new theme.txt to show the hotkey tips. You can also change the position and color.
```
+ hbox{ 
    left = 30%
    top = 95%
    width = 10%
    height = 25
    + label {text = "@VTOY_HOTKEY_TIP@" color = "blue" align = "left"} 
}
```

**Comment by megaenderdragon** [comment]
Thanks

It worked. can you please add an ability to change between wallpapers in ventoy? Because Im fed up with changing json and txt file extensions manually. I think the best solution will be to add function to right left arrow keys to change wallpapers

**Reply by ventoy** [reply]
This CI release add  `F5 Tools ---> Theme Select` menu.
https://github.com/ventoy/Ventoy/actions/runs/1466626280
https://www.ventoy.net/en/doc_github_ci.html

If you set more than one themes, you can swith between them with `F5 Tools --> Theme Select` menu.

**Comment by megaenderdragon** [comment]
Can you guide me on how to set it up? Should I name the files ventoy 1.json, ventoy 2.json etc? Or how

**Comment by ventoy** [comment]
You should use `theme plugin` to set more than one themes.
https://www.ventoy.net/en/plugin_theme.html

For example:
```
{
    "theme": {
        "file": [
            "/ventoy/themes/theme1/theme.txt",
            "/ventoy/themes/theme2/theme.txt",
            "/ventoy/themes/theme3/theme.txt",
            "/ventoy/themes/theme4/theme.txt"
        ]
    }
}

```

**Reply by megaenderdragon** [reply]
How to add more than 4 wallpapers? When copying the example and adding 2 more lines of "/ventoy..." it shows syntax error. IDK is this because I missed 2 "," or what?

**Reply by ventoy** [reply]
You should do some research about grub2 theme and json.
This option is to set theme not wallpapers.  wallpapers is part of theme. theme contains wallpapers/fonts/icons......

**Comment by megaenderdragon** [comment]
I meant to add more than 4 themes

śr., 17 lis 2021, 16:08 użytkownik longpanda ***@***.***>
napisał:

**Reply by megaenderdragon** [reply]
Ok. Issue solved. I needed to add 2 more comas (,)

**Comment by megaenderdragon** [comment]
One more think though, when will the CI theme changing be an integral part of the next release?

**Reply by ventoy** [reply]
Yes.

**Reply by megaenderdragon** [reply]
Will you also keep theme randomization? It can be as an entry: select random theme on boot

**Reply by ventoy** [reply]
Yes.
But it should be noted that:
`Which theme to use when boot` and `theme select` are two independent different features.
When you set more that one themes, you can choose to use a fix theme when boot or you can choose to randomly select a theme when boot.
But no matter how you choose, you can dynamically change the theme with `theme select` menu after boot.
But the `theme select` menu only change the theme for the current environment, it's temporary and can not be saved for next boot.
